src/append/opentelemetry.rs

```diff
@@ -46,20 +47,26 @@ pub enum OpentelemetryWireProtocol {
 #[derive(Debug)]
 pub struct OpentelemetryLog {
     name: String,
-    category: String,
     library: Arc<InstrumentationLibrary>,
     provider: LoggerProvider,
 }
 
 impl OpentelemetryLog {
     pub fn new(
         name: impl Into<String>,
-        category: impl Into<String>,
         otlp_endpoint: impl Into<String>,
         protocol: OpentelemetryWireProtocol,
+    ) -> Result<Self, opentelemetry::logs::LogError> {
+        Self::new_with_labels(name, otlp_endpoint, protocol, [])
+    }
+
+    pub fn new_with_labels(
+        name: impl Into<String>,
+        otlp_endpoint: impl Into<String>,
+        protocol: OpentelemetryWireProtocol,
+        labels: impl IntoIterator<Item = (Cow<'static, str>, Cow<'static, str>)>,
```

Comment:
These are public APIs. It'd be better we add comments to explain the constructor's arguments.